api/src/services/applications/create_application.py

```diff
@@ -77,6 +78,7 @@ def create_application(
         application_id=uuid.uuid4(),
         competition_id=competition_id,
         application_name=application_name,
+        application_status=ApplicationStatus.IN_PROGRESS,
```

Comment:
This was a small feature / ticket I missed making. It's a one-line change and I had to fix it to test the endpoints worked together anyways.